packages/strapi/lib/commands/generate-template.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const { resolve, join, basename } = require('path');
+const fse = require('fs-extra');
+const chalk = require('chalk');
+const inquirer = require('inquirer');
+
+// All directories that a template could need
+const TEMPLATE_CONTENT = ['api', 'components', 'config/functions/bootstrap.js', 'data'];
+
+async function createTemplate(templatePath) {
+  // Get path to template directory: strapi-template-<name>/template
+  const contentPath = join(templatePath, 'template');
+  let successMessage = 'create';
+
+  // Check if the correct template directory structure exists
+  const exists = await fse.pathExists(contentPath);
+  const templateBase = basename(templatePath);
+
+  if (exists) {
+    // Confirm the user wants to update the existing template
+    const inquiry = await inquirer.prompt({
+      type: 'confirm',
+      name: 'confirm',
+      message: `${chalk.yellow(templateBase)} already exists.  Do you want to replace it?`,
+    });
+
+    if (!inquiry.confirm) {
+      process.exit(0);
+    }
+
+    successMessage = 'update';
+  }
+
+  // Create/update the template
+  await fse.ensureDir(contentPath);
+  console.log(`${chalk.cyan(successMessage)}: ${templatePath}`);
```

Comment:
I think you can remove that log as we aren't really update a folder if it already exist. We could only display a final succss message after the copy at the end wdyt ?

Reply:
Sounds good to me!  I updated, and also added the json config file.